src/Controls/tests/UITests/Tests/Issues/Issue19556.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues;
+
+public class Issue19556 : _IssuesUITest
+{
+	public override string Issue => "[Android] Systemfonts (light/black etc.) not working";
+
+	public Issue19556(TestDevice device)
+		: base(device)
+	{ }
+
+    [Test]
+	public void SystemFontsShouldRenderCorrectly()
+	{
+		this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
+
+		_ = App.WaitForElement("label");
+
+		// The test passes if fonts are correctly rendered
+		VerifyScreenshot();
```

Comment:
Waiting CI to pass tests and generate the reference snapshot.

Reply:
@jsuarezruiz how do we get the screenshot? this looks ok to merge now otherwise.

Reply:
Right now, you need to go to the UITest run https://dev.azure.com/xamarin/public/_build/results?buildId=110953&view=ms.vss-test-web.build-test-results-tab&runId=2620968&resultId=100163&paneView=debug
And check the failed test . 
<img width="1178" alt="Screenshot 2024-03-13 at 16 17 21" src="https://github.com/dotnet/maui/assets/1235097/9cb53ff8-6533-40d7-890e-f0511f63f2ac">

Then go to the Attachments and download the image for the test SystemFontsShouldRenderCorrectly
<img width="718" alt="Screenshot 2024-03-13 at 16 18 18" src="https://github.com/dotnet/maui/assets/1235097/e01b63b3-9ee7-49fe-b31a-ed1923376afc">

Reply:
![SystemFontsShouldRenderCorrectly](https://github.com/dotnet/maui/assets/1235097/e174c67f-c337-4955-bfb5-361623250992)